docs/_includes/default.html

```diff
@@ -35,6 +35,17 @@ <h1>
     </header>
 
     <main id="content">{{ content }}</main>
+	
+	<aside>
```

Comment:
We should center the `<aside>` image in a different way than currently, because it makes the anchor go full page width, which makes a larger area clickable than probably intended. I'm adding a class here to be able to select and style it

```suggestion
	<aside class="sponsorship">
```